src/main/java/org/osiam/ResourceServer.java

```diff
@@ -67,11 +58,26 @@
     @Value("${org.osiam.resource-server.db.vendor}")
     private String databaseVendor;
 
+    public static void main(String[] args) {
+        SpringApplication.run(ResourceServer.class, args);
+    }
+
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
+        return application
+                .showBanner(true)
+                .sources(ResourceServer.class);
+    }
+
     @Bean
     public static PropertySourcesPlaceholderConfigurer properties() {
```

Comment:
I would like to use the [`@PropertyResource`](http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/context/annotation/PropertySource.html) annotation instead of creating this bean. But I need to set the hibernate naming strategy property, because Spring use it's [own](https://github.com/spring-projects/spring-boot/blob/master/spring-boot/src/main/java/org/springframework/boot/orm/jpa/hibernate/SpringNamingStrategy.java).